Sources/ImperialCore/Helpers/Optional+Imperial.swift

```diff
@@ -5,7 +5,7 @@ extension Optional {
     /// - Parameter error: The error to throw if the optional is `nil`.
     /// - Returns: The value contained in the optional.
     /// - Throws: The error passed in if the optional is `nil`.
-    public func value(or error: Error) throws -> Wrapped {
+    public func value(or error: any Error) throws -> Wrapped {
```

Comment:
I'm honestly not a fan of this, I'd much rather `guard` unwrap when needed, but I'm not really precious about it. What do you think?

Reply:
I totally agree